src/components/Header.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from "react";
+import styled from "styled-components";
+import {
+  space,
+  typography,
+  layout,
+  flexbox,
+  grid,
+  position,
+  SpaceProps,
+  TypographyProps,
+  LayoutProps,
+  FlexboxProps,
+  GridProps,
+  PositionProps,
+} from "styled-system";
+import NavMenu from "./NavMenu";
+import i18next from "i18next";
+import { NavLink } from "react-router-dom";
+
+const HeaderContainer = styled.div<
+  LayoutProps & SpaceProps & GridProps & PositionProps & FlexboxProps
+>`
+  width: 100%;
+  ${layout};
+  ${space};
+  ${grid};
+  ${position};
+  ${flexbox};
+`;
+
+const H1 = styled.h1<SpaceProps & TypographyProps & GridProps>`
+  ${space};
+  ${typography};
+  ${grid};
+`;
+
+const Container = styled.div<
+  FlexboxProps & GridProps & LayoutProps & SpaceProps & TypographyProps
+>`
+  background-color: black;
+  width: 100%;
+  ${flexbox};
+  ${grid};
+  ${layout};
+  ${space};
+  ${typography};
+`;
+
+const LangButton = styled.button<SpaceProps & TypographyProps>`
+  background: transparent;
+  border: none;
+  ${typography};
+  ${space};
+  transition: transform 0.2s;
+  transform-origin: left;
+  transform-origin: right;
+  &:hover {
+    transform: scale(1.05);
+`;
+
+const Header = () => {
+  const fontSizes = [0, null, null, null, null, 1, 4, null, null, 5];
+  const onLangClick = useCallback(
+    (countryId: string) => () => i18next.changeLanguage(countryId),
+    []
+  );
+
+  return (
+    <HeaderContainer
+      gridTemplateColumns={["repeat(3, 1fr)"]}
+      display="grid"
+      position="absolute"
+      top={6}
+      alignItems="center"
+      px={8}
+    >
+      <NavLink to="/">
+        <Container gridColumn={1}>
+          <H1 fontSize={fontSizes} style={{ zIndex: 2 }}>
```

Comment:
no inline styles, the `zIndex` can be moved to where the `H1` component is defined above on line 32

Reply:
the `zIndex` isn't actually working as its intended to can remove. 

essentially I wanted the big Oxymore logo in the top left of the `Homepage` component to "cover up" the little Oxymore as seen in the `Header`.

Reply:
gonna try a different approach and abstract the lang buttons into a separate component so not to bulk everything into the header. allows me to have a bit more control